### PR TITLE
Fixed a ton of tooltips popping up in 3d viewer

### LINKED ISSUE #51: Fix a ton of tooltips popping up in 3d viewer
https://discord.com/channels/1233487248129921135/1265314333764878387/1312925007156088872

Image

/bounty $8

### PR DESCRIPTION
**Pull request by Abse2001**
/claim #51
/closes #51 

https://github.com/user-attachments/assets/243928a5-384d-4210-a00d-b7b454a595c8

### CODE REVIEW

File: src/ContainerWithTooltip.tsx [hunk 1]
```diff
@@ -16,73 +16,60 @@ const ContainerWithTooltip = ({
...
+      lastValidPointRef.current = null
       onHover(null)
     },
+    [onHover],
+  )
+
+  const groupProps: GroupProps = {
+    onPointerEnter: handlePointerEnter,
+    onPointerMove: handlePointerEnter,
+    onPointerLeave: handlePointerLeave,
```
Comment (seveibar) [inline src/ContainerWithTooltip.tsx:69]: This is unconventional, can you just set the propd normally

**Review by seveibar — APPROVED** [review]
Note comment, unconventional patterns often get copied by ai or wayward controbutors

File: src/ContainerWithTooltip.tsx (hunk 1 above)
Comment (Abse2001) [reply src/ContainerWithTooltip.tsx:69]: Ok

**Comment by Abse2001** [comment]
@seveibar the problem is that ts don't see them as GroupProps if I don't spread them with GroupProps type while they still works it's a type error I couldn't figure out another way any ideas?
image

**Comment by seveibar** [comment]
@Abse2001 hmm ok. Its because the intrinsic-jsx is loading from tscircuit rather than react-three. The best solution would probably be to define an alias "ThreeGroup" class or something but its fine

**Pull request merged**